EasyCommands/Program.cs

```diff
@@ -149,10 +150,13 @@ public partial class Program : MyGridProgram {
                 var stateTuple = programStateMap[state];
                 Info(stateTuple.Key);
                 Runtime.UpdateFrequency = stateTuple.Value ? updateFrequency : UpdateFrequency.None;
-            } catch(Exception e) {
-                Print("Exception Occurred:");
+            } catch (Exception e) {
+                Print($"{(e is ParserException ? "Parser Exception" : e is RuntimeException ? "Runtime Exception" : "System Exception")} Occurred:");
                 Print(e.Message);
```

Comment:
nit: I might suggest "Parsing Exception" vs "Parser", and "Unknown" vs "System".  Not sure users will know the difference between Runtime & System.